src/components/ContactsList/ContactsList.jsx

```diff
@@ -32,13 +32,32 @@ const ContactsList = props => {
     if (name[0] !== lastLetter) {
       rows.push(<ContactHeaderRow header={name[0] || "EMPTY"} />);
     }
-    rows.push(<ContactRow key={contact._id} contact={contact} />);
+    rows.push(
+      <ContactRow
+        key={contact._id}
+        contact={contact}
+        selection={
```

Comment:
Why not 2 distinct props here (`selected` and `onSelect`)?

Reply:
because I would like to stand on one of this to display or not the `<ContactSelection />` component. I am sure I can do best, but not after one tiring day.